### DISCUSSION TITLE
What is the license of the code and the content?

### DISCUSSION THREAD

**Discussion by Trilarion** [ekolis/FrEee#190] [Q&A] [answered]
I could not find any license information in the code directly or somewhere else in the repository. What is the license of the code and the content?

**Comment by ekolis** [comment, marked as answer]
The license is one of the Creative Commons ones, I don't remember which. I think it's in a file in the Licenses folder in the repo. I've heard that CC isn't good for code but another contributor insisted on it saying it would make it easier to use CC licensed art or something...

**Comment by Trilarion** [comment]
Thank you for the reply. In the [credits.txt](https://github.com/ekolis/FrEee/blob/master/FrEee.Assets/Licenses/credits.txt) file the license is named as "a Creative Commons Attribution Noncommercial License" and probably means the license in [license.txt](https://github.com/ekolis/FrEee/blob/master/FrEee.Assets/Licenses/license.txt).

Upon comparison with available CC licenses I see that the license in "license.txt" is [version 2.0 of CC BY-NC](https://creativecommons.org/licenses/by-nc/2.0/legalcode) and while it's not often used for code it probably can be used and means the artwork and the code is both under the CC-BY-NC-2.0 license. 

What people usually do is using a variant of the CC for the artwork and another license like GPL or MIT or similar for the code.

**Reply by ekolis** [reply]
Yeah, I wanted to use an MIT or BSD license or something, but someone else was adamant for some reason that CC would be better...

**Comment by Trilarion** [comment]
The main difference in this case is the non-commercial clause. MIT or BSD allow commercial use. Even GPL does allow with the caveat that you have to publish the sources so that typically reduces the commercial value. The license here completely forbids it. And that may actually backfire. For example even if you would distribute the game for free basically but put advertisement on the download page that might already be seen as commercial activity. For more information see for example: https://blog.okfn.org/2013/01/08/consequences-risks-and-side-effects-of-the-license-module-non-commercial-use-only-2/ or https://wiki.creativecommons.org/wiki/NonCommercial_interpretation. Not that it changes much. The license is what it is and cannot be changed without consent of all authors anyway, so mostly is set.

For a bit of context. I gather information about open source games under suitable licenses and create a website showing these information (https://github.com/Trilarion/opensourcegames). Part of it is checking the license status and that's why I asked here.

Thanks again for the very quick answer. Good luck with the project. At some point I might try it out. I'm a big fan of strategy space games but haven't much experience with C#. In any case I know that it's a lot of work (each of these projects).
